calibration/step-5-unify_measurements.py

```diff
@@ -66,6 +81,9 @@ def unify_measurements(
         unified_json = json.load(json_file)
         unified_json["LocalRank"] = group_index if groups_num != 1 else -1
 
+    moe_experts_data = {}
+    expert_num = max([analyze_expert_name(i)[1] for i in unified_json["Nodes"] if is_moe_experts(i)]) + 1
```

Comment:
For the model without `moe`, this list is empty; please make sure this case still works.